### DISCUSSION TITLE
[Spring MVC] 초록 스터디 Step1 회고

### DISCUSSION THREAD

**Discussion by pykido** [BDD-CLUB/challenge-every-week-posting#439] [5. 회고]
[글 바로가기](https://animoto1.tistory.com/entry/Spring-MVC-%EC%B4%88%EB%A1%9D-%EC%8A%A4%ED%84%B0%EB%94%94-Step1-%ED%9A%8C%EA%B3%A0)

- 이제야 올리네요 ㅠㅠㅠ. step2랑 알고리즘 푼 것도 빠른 시일 내에 정리해서 올려볼게요!

**Comment by shkisme** [comment]
와 정리 너무 잘 해놓으셨네요 👍👍
